### PR TITLE
Add Scalar Quantization pipeline

### PR DESCRIPTION
**Pull request by trengrj**
- Use more accurate name quantization and allow configurable env var `QUANTIZATION` than can be `pq` or `sq`.
- Update to newest python client
- Add Scalar Quantization pipelines for cosine and distance distances

### CODE REVIEW

File: apps/ann-benchmarks/run.py [hunk 1]
```diff
@@ -36,15 +36,15 @@
 
 
 parser = argparse.ArgumentParser()
-client = weaviate.connect_to_local()
+client = weaviate.connect_to_local(skip_init_checks=True)
```
Comment (donomii) [inline apps/ann-benchmarks/run.py:39]: The commit says temporarily, will this be removed in another patch?

Comment (trengrj) [reply apps/ann-benchmarks/run.py]: Thanks @donomii, I've removed this was used for testing.
